examples/platform/telink/common/src/AppTaskCommon.cpp

```diff
@@ -33,6 +33,10 @@
 #include "OTAUtil.h"
 #endif
 
+#ifdef CONFIG_CHIP_ICD_SUBSCRIPTION_HANDLING
+#include <app/InteractionModelEngine.h>
```

Comment:
Needs to include ICDUtil.h too, right?

Reply:
@bzbarsky-apple Moved ICDUtil.h include into cpp file